src/test/java/dev/farhan/movieist/movies/Movie_Movie_d09d4293c0_Test.java

```diff
@@ -0,0 +1,67 @@
+package dev.farhan.movieist.movies;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "movies")
+public class Movie_Movie_d09d4293c0_Test {
+
+    @Id
+    private ObjectId id;
+    private String imdbId;
+    private String title;
+    private String releaseDate;
+    private String trailerLink;
+    private String poster;
+    private List<String> backdrops;
+    private List<String> genres;
+    private List<Review> reviews;
+
+    @Test
```

Comment:
Error: Test cannot be resolved to a type.
Reason: Missing imports import org.junit.jupiter.api.Test;